ui/src/js/components/shared/modals/LoginPrompt.js

```diff
@@ -31,11 +65,21 @@ export default class LoginPrompt extends Component {
             <div className="LoginPrompt">
               <div>
                 <p>Your authentication token has expired and must be renewed to perform this action.</p>
```

Comment:
maybe simplify message:

```suggestion
                <p>Your session has expired and must be renewed to perform this action.</p>
```